base/uk.ac.stfc.isis.ibex.opis/resources/opi_info.xml

```diff
@@ -2916,6 +2916,27 @@
         </categories>
       </value>
     </entry>
+    <entry>
+      <key>Automatic Needle Valve Controller</key>
+      <value>
+        <type>UNKNOWN</type>
```

Comment:
I think temperature controller or eurotherm would be better than `UNKNOWN` here.